lib/paperclip/has_attached_file.rb

```diff
@@ -100,6 +101,20 @@ def add_active_record_callbacks
       end
     end
 
+    def add_background_processing
+      require "paperclip/process_job"
+      name = @name
+      if @klass.respond_to?(:after_commit)
```

Comment:
Should I care for case when there is no `after_commit` and add `after_save` then?